redxouls/Echo
# Echo-#78 add respawn point

### What is this PR for?

add respawn point

### What type of PR is it?
[Feature]

### Todos
* [x] - add respawn point
* [x] - change the respawn button's image  

### What is the Github issue?

[ECHO-#78](https://github.com/redxouls/Echo/issues/78)
### How should this be tested?

* dead before finish tutorial, and check if player will respawn at origin
* dead after finish tutorial, and check if player will respawn at altar
### Screenshots (if appropriate)

### Questions:
* Do the license files need updating? No
* Are there breaking changes for older versions? No
* Does this need new documentation? No
